doc/en/apiref.rst

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
It looks like this file got moved on the filesystem rather than "git mv"ed - was there a reason for that?